src/AppKit/NSSliderTouchBarItem.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+
+namespace XamCore.AppKit {
+
+	public partial class NSSliderTouchBarItem {
+		// If you modify, also search for other copies
+		NSObject target;
```

Comment:
If we are going to give guidance on "Search for other copies", perhaps we should use a unique string to identify this, like "MustBeIdenticalXXX"